website/DEPENDENCIES.md

```diff
@@ -0,0 +1,74 @@
+# Dependencies
+
+Generated by `npx license-checker --producation --excludePrivatePackages`:
```

Comment:
Use `npx license-checker --production --excludePrivatePackages --csv` to omit user specific path. And we can add a CI test.

Reply:
To be concrete, this command to be inlined in the `README.md`, and we gen a `DEPENDENCIES.node.csv` and then re-gen it in CI and check the diff.

Reply:
DONE.

Reply:
Unresolve. Please check the comment above once more.

Reply:
I'm not satisfied with license-checker either. How about merge this PR as is and polish this part later.

Reply:
Ok, let me do this.

Reply:
Added.